docs/source/Intro.rst

```diff
@@ -24,6 +24,13 @@ Challenges for accurate weather predictions still exist in numerical models. To
     - Larger TC False Alarm Rate in the western North Atlantic
     - Exacerbated low instability bias, which is largely related to dry soil moisture
     - Poor representation of radiation inversions
+  .. group-tab:: RRFS.v1alpha
+
+    - Better hurricane track forecasts for strong storms in the Atlantic basin
+    - Weak hurricane intensity forecasts
+    - Unstable signal over mountain area
+    - Slightly better cold air damming signal
```

Comment:
When listing biases (of SRW and MRW), avoid saying _better_ or _worse_ since the reader does not know better/worse with respect to what.

Reply:
Thanks, Ligia! It's updated.